src/main/kotlin/com/linktime/plugin/jira/insightobjectgraph/impl/objects/ListObjectsEndpoint.kt

```diff
@@ -40,8 +40,11 @@ constructor(private val listService: ListObjectsService) {
                        @QueryParam("schemaId") schemaId: Int?,
                        @QueryParam("attributes") attributes: Set<String>,
                        @QueryParam("resolve_attributes") resolveAttributes: Set<String>,
-                       @QueryParam("resolve_relations") resolveRelations: Boolean): Response {
-        val siteObjects = listService.getObjects(objectTypeName, schemaId, attributes, resolveAttributes, resolveRelations)
+                       @QueryParam("resolve_relations") resolveRelations: Boolean,
+                       @QueryParam("iql_string") iqlString: String): Response {
```

Comment:
The iql_string parameter is optional. If I call the endpoint without the parameter, will it actually be bound here to the empty string automatically, or will it be null?

Reply:
Updated it to nullable, it supposedly should have worked before though, as most of the QueryParameters are Optional but only 'schemaId' is nullable